src/GovUk.Education.ExploreEducationStatistics.Admin/Services/FileUploadsValidatorService.cs

```diff
@@ -60,7 +60,7 @@ public class FileUploadsValidatorService : IFileUploadsValidatorService
 
         public async Task<Either<ActionResult, Unit>> ValidateFileForUpload(IFormFile file, FileType type)
         {
-            if (type != Ancillary && type != Chart && type != Image)
+            if (type is FileType.Data or Metadata)
```

Comment:
Should this include all the `FileType`'s except for `Ancillary`, `Chart` and `Image`?